src/commands/createFunction/openAPISteps/OpenAPICreateStep.ts

```diff
@@ -109,3 +132,33 @@ async function addAutorestSpecificTypescriptDependencies(context: IFunctionWizar
         return data;
     });
 }
+
+async function addAutorestSpecificPythonDependencies(context: IFunctionWizardContext): Promise<void> {
+
+    let oldRequirements: string = '';
+    let newRequirements: string = '';
+    const requirementsPath: string = path.join(context.projectPath, requirementsFileName);
+
+    // Current list of dependencies to be added to support stencil generated projects.
+    const dependenciesToAddArray: string[] = ['msrest'];
+
+    // Some custom requirements might not have newline character added already.
+    const dependenciesToAdd: string = `\n${dependenciesToAddArray.join('\n')}\n`;
+
+    if (await fse.pathExists(requirementsPath)) {
+        try {
+            oldRequirements = (await fse.readFile(requirementsPath)).toString();
+            newRequirements = oldRequirements.concat(dependenciesToAdd);
+        } catch (error) {
+            // If we failed to parse or edit the existing file, just ask to overwrite the file completely
```

Comment:
I think you copied this from `confirmEditJsonFile`, but it doesn't really make sense in this case. We're not parsing or editing JSON, so there's way fewer chances for problems. The only real error could happen from reading the file, but if that fails we probably wouldn't be able to write to it either. I would get rid of the whole catch block.

Reply:
Done. Just added a warning log to make sure the user is aware of the issue and takes action.